modules/fee/handler.go

```diff
@@ -11,76 +11,67 @@ import (
 // NameFee - namespace for the fee module
 const NameFee = "fee"
 
-// AccountChecker - interface used by SimpleFeeHandler
-type AccountChecker interface {
-	// Get amount checks the current amount
-	GetAmount(store state.KVStore, addr basecoin.Actor) (coin.Coins, error)
-
-	// ChangeAmount modifies the balance by the given amount and returns the new balance
-	// always returns an error if leading to negative balance
-	ChangeAmount(store state.KVStore, addr basecoin.Actor, coins coin.Coins) (coin.Coins, error)
+// Bank is a default location for the fees, but pass anything into
+// the middleware constructor
+var Bank = basecoin.Actor{App: NameFee, Address: []byte("bank")}
```

Comment:
I feel like the default for fee handling should be to slash the fees. Is the banker supposed to be a location for the fees to accumulate too? Can be coins be sent from this account if it was associated with a pubkey? - This is a bit weird and confusing